scoring.js

```diff
@@ -0,0 +1,83 @@
+const attempts = 0
+const completions = 0
+const passingYards = 1 / 25
+const touchdowns = 6
+const interceptions = -3
+const rushingYards = 1 / 10
+const fumbles = -3
+const receptions = 1
+const receivingYards = 1 / 10
+const returnYards = 1 / 15
+const returns = 0
+module.exports = calculateScore
+
+function passingStats(passing) {
+  let passingTotal = (passing.completions * completions)
```

Comment:
These can be `const`, you're not re-assigning the variable anywhere. Also if you're gonna chain the math together then you can just return, no need for a variable at all. Also if you're gonna chain, it's convention to put the operator at the end of the line as an indicator that the line continues. Also completions is not something they get points for so putting that in your calculation just adds unneeded code.

```JavaScript
return (passing.yards / 25) +
  (passing.touchdowns * 6) +
  (passing.interceptions * -3)
```

However, since this is several calculations it might be worth doing them separately and then adding them together.

```JavaScript
function calculatePassingScore(passing) {
  const yards = passing.yards / 25
  const touchdowns = passing.touchdowns * 6
  const interceptions = passing.interceptions * -3

  return yards + touchdowns + interceptions
}
```